src/main/java/edu/cornell/kfs/module/purap/rest/application/PurchasOrderApplication.java

```diff
@@ -0,0 +1,25 @@
+package edu.cornell.kfs.module.purap.rest.application;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+import edu.cornell.kfs.module.purap.rest.resource.PurchaseOrderResource;
+
+@ApplicationPath("ws/purapp/po")
+public class PurchasOrderApplication extends Application {
```

Comment:
There is a typo in this class name

Reply:
thanks, changed to PurchaseOrderApplication